python/pytests/golden/hash_test/test_hash_integer.jsonl

```diff
@@ -0,0 +1,6 @@
+{"_time":"2021-01-01T00:00:00.000000000","_key":"Ben","m":5.0,"hash_m":16461383214845928621}
+{"_time":"2021-01-01T00:00:00.000000000","_key":"Ryan","m":8.0,"hash_m":6794973171266502674}
+{"_time":"2021-01-02T00:00:00.000000000","_key":"Ryan","m":9.0,"hash_m":15653042715643359010}
+{"_time":"2021-01-03T00:00:00.000000000","_key":"Ben","m":8.0,"hash_m":6794973171266502674}
+{"_time":"2021-01-04T00:00:00.000000000","_key":"Ben","m":null,"hash_m":0}
+{"_time":"2021-01-04T00:00:00.000000000","_key":"Ryan","m":9.0,"hash_m":15653042715643359010}
```

Comment:
these values aren't what I expected... old docs show different values:
 https://kaskada.io/docs-site/kaskada/main/fenl/catalog.html#hash

also I wasn't expecting `null` to hash to `0`

Reply:
The behavior of hash has changed, but the catalog hasn't been updated since the last time we pushed to read the docs from the old kaskada repo (so basically, really old). The *content* in the repo is likely more up to date:

https://github.com/kaskada-ai/kaskada/blob/main/crates/sparrow-catalog/catalog/hash.toml#L1

We switched to using a hash function based on DataFusion which let us hash structs and things, but it changed the results when applied to `null` to return `0`.